.travis.yml

```diff
@@ -34,13 +53,32 @@ jobs:
       name: "Storybook a11y testing"
     - script: yarn test:backend:core:testdbsetup && yarn test:backend:core
       name: "Backend unit tests"
+    - name: "Cypress 1"
+      env:
+        - STAGE_NAME="4x-electron on Travis CI"
+      <<: *defaults
```

Comment:
Can you help me understand what these four lines do?

Reply:
Actually sorry: what I'm curious about is why we have multiple Cypress 1, 2, 3, 4, but all with the same STAGE_NAME env variable

Reply:
It's supposed to create a job that has the same STAGE_NAME variable defined for the script, which would run Cypress in parallel (hopefully splitting up the tests across 4 jobs) to reduce test time.

Reply:
The syntax for using the default script is found in this example: https://github.com/cypress-io/cypress-example-kitchensink/blob/master/.travis.yml. But I couldn't find any other documentation on it.

Reply:
Cool! Hope it works? (And: want to include the `# run tests in parallel by including several test jobs with same name variable` comment in this PR too?)

Reply:
So the parallel testing didn't work out (seems like we'd need a special Cypress key); so I didn't include the comment.